spec/where_spec.cr

```diff
@@ -0,0 +1,93 @@
+require "./spec_helper"
+
+describe LuckyRecord::Where do
+  describe "Equal" do
+    describe "#negated" do
+      it "returns a NotEqual" do
+        LuckyRecord::Where::Equal.new("column", "value").negated
```

Comment:
All of those `LuckyRecord::Where` tests feel super boilerplate-y and I was unable to verify the name/column instance vars. Thats why it's a `.should be_a`. Do you have any ideas to improve that?

Reply:
Also, at the moment we could get away with not having any of that, since `Queryable` only uses `Equal` ... hmmm

Reply:
I think Queryable will work with all of them. I pulled it down and tried with gt and lt and those all worked. Which I think is cool that it automatically works like that. I agree these tests do seem a bit boilerplate. There should be a `column` and `value` getter defined on the clause so I think you could do something like this:

```crystal
clause = LuckyRecord::Where::NotEqual.new("column", "value").negated
clause.column.should eq "column"
clause.value.should eq "value"

# And since they are all pretty similar, I think there could be just one "it" block and a helper method could be defined

it "negates the clause" do
  should_negate LuckyRecord::Where::NotEqual, with: LuckyRecord::Where::Equal
  should_negate LuckyRecord::Where::Like, with: LuckyRecord::Where::NotLike
end

private def should_negate(original_where, with expected_negated_where)
  clause = original_where.new("column", "value").negated
  clause.column.should eq "column"
  clause.value.should eq "value"
  clause.should be_a(expected_negated_where)
end
```